<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSampler</class>
 <widget class="QWidget" name="frmSampler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>采样控制</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#frm{
background-image: url(:/image/main2.jpg);
}

QLabel{
color:rgba(255,255,255,255);
}

QGroupBox{
color:rgba(0,255,0,255);
}</string>
  </property>
  <widget class="QWidget" name="frm" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="gbox_sampler">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>采样控制</string>
    </property>
    <widget class="QComboBox" name="comboBox_SampleStartBottle">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>开始瓶号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>采样协议：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSampleFlow">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>采样量：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_sampleMode">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>SBC-6000型</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>北京环科采样器</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>ml</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_StartSampling">
     <property name="geometry">
      <rect>
       <x>649</x>
       <y>70</y>
       <width>124</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>启动采样</string>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_setsave">
     <property name="geometry">
      <rect>
       <x>649</x>
       <y>110</y>
       <width>124</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>保存配置</string>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Cancel">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>124</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>230</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>ml</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbox_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>781</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>采样瓶状态</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>2号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>3号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>4号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>5号瓶</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_GetSamplerStatus">
     <property name="geometry">
      <rect>
       <x>653</x>
       <y>20</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>获取状态</string>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_CleanSamplerStatus">
     <property name="geometry">
      <rect>
       <x>653</x>
       <y>60</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清空状态</string>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>7号瓶</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>8号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_8">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>6号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>10号瓶</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>9号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>12号瓶</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>13号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_12">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_15">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_13">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>11号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_11">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>190</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>15号瓶</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>14号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_14">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>140</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>17号瓶</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>110</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>18号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_17">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_20">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>180</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_18">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>16号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_16">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>190</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>20号瓶</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>150</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>19号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_19">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>140</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>22号瓶</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>110</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>23号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_22">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>60</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_23">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>100</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>21号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_21">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>150</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>24号瓶</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottle_status_24">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>140</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
